edumfa/lib/policy.py

```diff
@@ -821,11 +824,8 @@ def _policy_matches_request_header_condition(policy, key, comparator, value, req
                     raise PolicyError(
                         "Invalid comparison in the HTTP header conditions of policy {!r}".format(policy['name']))
             else:
-                log.warning("Unknown HTTP header key referenced in condition of policy "
```

Comment:
Why are theses messages removed?

Reply:
The warning was moved to the function `_react_to_unset`.
The warning is only logged when the new functionality is not configured or set to `exception`